docs/cloud/cloud-authentication/cloud-auth-manage.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Manage API Keys
+layout: default
+parent: Cloud
+has_children: true
+nav_order: 5
+---
+
+# How do I manage API keys in FeatureBase Cloud?
+{: .no_toc }
+
+This page provides an overview of FeatureBase API keys and links to guide you through the process of managing API keys.
+
+FeatureBase API keys are used for authentication [when accessing FeatureBase programmatically](/docs/cloud/programmaticaccess/).
+
+## Before you begin
+{: .no_toc }
+{% include /cloud/cloud-before-begin.md %}
+
+{% include page-toc.md %}
+
+## Accessing API keys
+
+To view your API keys, click **Configuration** > **Manage API Keys**. Here you can see all of the API keys that are available to you. If you are an admin in your organization, you will be able to see all of the API keys in your organization (toggleable in the top right).
```

Comment:
Procedures that include menu items and actions should be setup as procedures. And in general the headings for procedures will be questions in the form "How do I <do the thing>?"

Here's an example of what I mean:

```
## How do I access API keys for my organization?

Administrator users can access organization API keys via the Cloud UI. 

1. Click **Configuration**
2. Click **Manage API Keys** 

## How do I find my API key secret?

FeatureBase Cloud does not store API Key secrets.

{: .warning}
>Organizations should revoke the API key if the secret is lost.
>* [Learn how to revoke an API key](/docs/cloud/cloud-authentication/cloud-auth-revoke-key/)

```